Terraform.sublime-syntax

```diff
@@ -319,6 +344,43 @@ contexts:
     - meta_content_scope: source.terraform
     - include: string-interpolation-body
 
+  ipv4-at-start:
+    - match: \G{{ipv4}}\b
```

Comment:
`\G` means it matches after any previously consumed token, i.e. you could have multiple IPs within a single string or have an IP after an escape sequence. A more consistent approach would be to have two different string body contexts where the IP patterns are only applied for the very first character and any other following character switches to the other string context.